core/controllers/domain_objects_validator.py

```diff
@@ -246,7 +246,7 @@ def validate_suggestion_images(files):
 
 
 def validate_exploration_change(exp_change_dict):
```

Comment:
I don't see a change to the function name (per [this comment](https://github.com/oppia/oppia/pull/16065#discussion_r979367986))?

Reply:
I have updated the name, though I'm not sure if it seems okay. Please let me know if I have to change.
Thanks